### PR TITLE
Privately remind users not to leave :+1: and :-1: as comments

### PR DESCRIPTION
**Pull request by bfred-it**
Less of a feature, more of a community service. This only appears to who left the comment.

Feature annoyance rating: 🔥🔥🔥💦💦

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -112,6 +112,7 @@
 	"stylelint": {
 		"extends": "stylelint-config-xo",
 		"rules": {
+			"selector-type-no-unknown": null,
```
Comment (bfred-it) [inline package.json]: @sindresorhus what do you say about this rule? Should it be configured to ignore customElements or should each repo have its own list of known types like it happens for JavaScript "`globals`"?

```
 51:46  ✖  Unexpected unknown type selector "g-emoji"  selector-type-no-unknown
```

**Comment by bfred-it** [comment]
I limited it to +1 and -1 because they are the most useless+common ones… but yeah they can probably be added too if accepted.

**Comment by hkdobrev** [comment]
We can use the image if the conversation is locked or there's a cool down period enabled in the settings repo 😆

**Comment by bfred-it** [comment]
That’d be hilarious actually 😂

**Comment by yakov116** [comment]
@bfred-it :+1:

**Comment by sindresorhus** [comment]
Wouldn't it be better to warn the user upfront? So if they only have a :+1: and :-1: and press "Comment", we would show a confirm dialog where we recommend using reactions instead? I was first thinking just blocking comments with 👍, but there are valid reasons to comment that sometimes.

**Comment by bfred-it** [comment]
Two reasons:
- JavaScript features can be disabled
- This one just annoys a bit, but doesn’t require clicking YES when posting (which could still be blindly accepted anyway)

**Comment by bfred-it** [comment]
Merging this since it looks like we're on the same page. For additional features, PRs welcome! :)

**Comment by wopian** [comment]
This message shows up for any comment that contain `:+1:` shortcodes _anywhere_ in its body.

It doesn't make sense to warn me about its usage in https://github.com/chrislgarry/Apollo-11/pull/323#issuecomment-374664335 for example.

**Comment by yakov116** [comment]
Implemented by github!

image

https://blog.github.com/changelog/2018-12-07-suggest-leaving-a-reaction/

**Pull request merged**